middlewares/src/main/java/com/jinyframework/middlewares/cors/Cors.java

```diff
@@ -136,20 +165,41 @@ private static void handleActual(Context ctx, Config config) {
         }
     }
 
-    private static boolean isAllowedOrigin(String origin, Config config) {
-        return config.allowOrigins.stream().anyMatch(org -> org.equalsIgnoreCase(origin));
+    private static boolean isAllowedMethod(String method, List<String> allowMethods) {
+        for (String am : allowMethods) {
```

Comment:
Can it be rewritten to `val am : allowMethods`

Reply:
Btw, IMHO we should use `lombok.var` and `lombok.val` where possible, which will make our code more consistency, I'll add this to the Developer Guide